lumigator/python/mzai/backend/backend/services/experiments.py

```diff
@@ -196,6 +200,48 @@ def create_experiment(
 
         return ExperimentResponse.model_validate(experiment_record)
 
+    def _get_experiment_jobs(self, experiment_id: UUID):
+        records = self._job_service.job_repo.get_jobs_by_experiment_id(experiment_id)
```

Comment:
You've added the `get_jobs_by_experiment_id` method in the `JobResultRepository` class and you're trying to invoke it using a `JobRepository` instance. This doesn't work:

```
AttributeError: 'JobRepository' object has no attribute 'get_jobs_by_experiment_id'
```

I guess you'd want to move the `get_jobs_by_experiment_id` method in the `JobRepository` class.

Reply:
I've opted to put this in the experiment repo since it's logically related to experiments and we don't instantiate a jobs repo in the experiment service - let me know what you think